frontend-cert/html-projects/travel-agency-page/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should have an `h1` element with the text `Discover Italy`.
+
+2. You should have a paragraph below the `h1` introducing the travel opportunities.
+
+3. You should have an `h2` element with the text `Packages`.
+
+4. You should have a `p` element introducing briefly the various packages.
+
+5. You should have an unordered list element with two list items. The two list items should have the text `Group Travels` and `Private Tours`, respectively. The text of each list item should be enclosed by an anchor element with a dead `href` attribute as placeholder, and a `target` attribute of `_blank`.
```

Comment:
This one feels a little long to me - but I guess it's fine. Was wondering if it should be broken up into two.

I would lean towards using a real link and not a dead one. Maybe we can make some dummy pages or something, or let users put whatever in them.

Reply:
Maybe it  would be better if we can make them link to a dummy page :+1: